extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/pages/dashboard/services/MiwConfigService.java

```diff
@@ -42,4 +41,14 @@ public DashboardMiwConfig buildMiwConfigOrNull() {
     String configValue(String configKey) {
         return config.getString(configKey, "");
     }
+
+    /**
+     * Maps a {@literal CONFIG_KEY} to {@literal config.key}
+     *
+     * @param envVarKey {@literal CONFIG_KEY}
+     * @return {@literal config.key}
+     */
+    public static String configKey(String envVarKey) {
```

Comment:
This duplicates the logic in https://github.com/sovity/edc-ce/pull/1051/files#diff-c5d05a8b16429350f436d4a2c8be84d45d4839a9e5cf72436991f75cca182e43R112

Would be nice to have it only once in some utils and give it a better name like `fromEnvCaseToPropertyCase`

Reply:
Also this version is broken because of the stoopid `split()` implemention from my PoV:

```java
    public static void main(String[] args) {
        System.out.println(configKey("__A__"));
    }
```
-> `..a`

Reply:
This is legacy code that will disappear after the next migration, and the props for the next migrations will be added to ConfigProp.